protecode/model.py

```diff
@@ -68,13 +68,15 @@ def cve(self):
     def cve_severity_str(self):
         return str(self.raw.get('vuln').get('cvss'))
 
+    def has_triages(self):
+        return self.raw.get('triage') is not None
```

Comment:
please check whether "triage" might also mean: "yes, it is really bad"

Reply:
also: please check if this check is specific enough.

by which I mean: if protecode app A contains vulnerability foo and is triaged "only for app A" (i.e. protecode still shows the same vulnerability for an app B, even after the assessment), then this check must not return True for component B.